src/resolvers/default.rs

```diff
@@ -163,6 +174,45 @@ impl Dh for Dh25519 {
     }
 }
 
+impl Dh for Dh448 {
+    fn name(&self) -> &'static str {
+        "448"
+    }
+
+    fn pub_len(&self) -> usize {
+        56
+    }
+
+    fn priv_len(&self) -> usize {
+        56
+    }
+
+    fn set(&mut self, privkey: &[u8]) {
+        copy_slices!(privkey, &mut self.privkey);
+        self.pubkey = x448::x448_unchecked(self.privkey, x448::X448_BASEPOINT_BYTES);
```

Comment:
What's the difference between this and the checked version? It looks like from the documentation that checked will assert that low-order points aren't used?

Reply:
Yep that’s completely correct. I used the unchecked version for consistency between X25519 and X448